tests/test_kerberos_client.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from src.kerberos_client import KerberosClient
+from shared_server import *
+
+
+@pytest.fixture
+def client():
+    return KerberosClient()
+ 
+def test_version(client):
+    assert client.version == 24
+
+def test_client_functionality(client):
```

Comment:
I think we should separate the tests per action meaning one method for register and one for sending a message